phase1/gce/gce.jsonnet

```diff
@@ -77,6 +93,20 @@ function(cfg)
           source_ranges: ["0.0.0.0/0"],
           target_tags: ["%(cluster_name)s-master" % p1],
         },
+        [names.master_internal_firewall_rule]: {
```

Comment:
Hmm, I'd be concerned about the stability of this IP. We rely on having a stable master IP for all our PKI stuff, changing this IP means we have to regenerate lots of TLS certs. This will probably need to be made to use the static IP once we support upgrades but it's OK for now.